app/src/main/java/com/fatalzero/rsandroidfinal_task/presentation/Fauvorite/AddViewModel.kt

```diff
@@ -11,19 +11,19 @@ import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 class AddViewModel @Inject constructor(
-    var saveUseCase: JokeSaveUseCase,
-    var jokeGetUseCase: JokeGetUseCase
+    var saveUseCase: com.fatalzero.rsandroidfinal_task.domain.usecase.JokeSaveUseCase,
```

Comment:
Please check here and in other places for right import

Reply:
done